mne/decoding/transformer.py

```diff
@@ -172,6 +172,34 @@ def transform(self, epochs_data, y=None):
         return _sklearn_reshape_apply(self._scaler.transform, True,
                                       epochs_data, y)
 
+    def fit_transform(self, epochs_data, y=None):
+        """Fit to data, then transform it.
+
+        Fits transformer to epochs_data and y and returns a transformed version 
+        of epochs_data.
+
+        Parameters
+        ----------
+        epochs_data : array, shape (n_epochs, n_channels, n_times)
+            The data.
+        y : None | array, shape (n_epochs,)
+            The label for each epoch.
+            Defaults to None.
+
+        Returns
+        -------
+        X : array, shape (n_epochs, n_channels, n_times)
+            The data concatenated over channels.
+
+        Notes
+        -----
+        This function makes a copy of the data before the operations and the
+        memory usage may be large with big data.
+        """
+        return _sklearn_reshape_apply(self._scaler.fit_transform, True,
+                                      epochs_data, y)
```

Comment:
why not just returning

self.fit(X, y).transform(X)

is this method covered by a test?